ebmlite/cli.py

```diff
@@ -0,0 +1,137 @@
+import contextlib
+import sys
+from xml.etree import ElementTree as ET
+
+import ebmlite
+
+import argparse
+import os.path
+from xml.dom.minidom import parseString
+
+
+###############################################################################
+# Utilities
+
+def errPrint(msg):
+    sys.stderr.write("%s\n" % msg)
+    sys.stderr.flush()
+    exit(1)
+
+
+@contextlib.contextmanager
```

Comment:
Using a context here seems overly complex; I'm not sure it adds anything. There's only one schema per run of the script, one input file, and at most one output file. The context manager is cool but I think it makes things harder to read.

Reply:
failing to clean up file handles is a bug, and it's not immediately identifiable (esp. with our tests). I don't trust myself to remember to explicitly put in a proper exception-proof cleanup step in every function where I use this because I am, on the whole, pretty error-prone. Context managers are the safest guarantee I have for proper file handle management. If you want to change it, I will leave it to you to do so safely and correctly.